docs/thesaurus/index.md

```diff
@@ -44,47 +59,49 @@ Within the `web/thesauruses/_meta` directory, create the new file you just named
 
 Within this file you will need to edit the following items to make the file work:
 
-* Leave the meta info alone (since it's a meta data file anyway)
-* Edit the `"categories"` section (but not the word "categories") to reflect the concept you are adding and as many ways as you can think of to show examples of how that concept is implemented in any hypothetical programming language. Don't worry if you can't think of them all, as more languages are added, people will add to this file.
-* Under `"categories"`, you will implement the basic structure for the JSON file to have those implementations. So the section will be the name of the concept ID you created in `meta_info.json`, then the sections within that will match the `categories` section exactly. You can look at some other meta files for how this looks.
+* Replace `"structure_id"` by the files name without the ".json" file ending.
+* Replace `"Structure Name"` with a friendly name for the structure, the same you gave it in `meta_info.json`
+* Edit the `"Category Name"`, ... fields to reflect the structure you are adding and add as many categories as you can think of that subdivide the structure you're adding in any hypothetical programming language.
+* Within every `"category"` block add as many concepts as you can think of to show examples of how that concept is implemented in any hypothetical programming language.
+
+Don't worry if you can't think about all concepts for all languages because as more languages are added people will add to this file, too.
 
-If you are adding a concept to all of Code Thesaurus, please make a pull request for this _separately_ from a language's implementation of this. This is to make sure the file can be reviewed first before languages try to implement the changes first. 
+If you are adding a concept to all of Code Thesaurus, please make a pull request for this _separately_ from a language's implementation of this. This is to make sure the file can be reviewed first before languages try to implement the changes.
 
 ## Add on a New Language
 
 If a programming language doesn't exist yet in Code Thesaurus, you are welcome to add it! There are a few steps to this process.
 
 First, you will need to open up `web/thesauruses/meta_info.json`. Within that file, the first section is for languages. Add in your language to that section in alphabetical order. From there, assign it to a directory name in the JSON structure.
 
-After you save that file, go make a new subdirectory with same name in `web/thesauruses/`. From there, copy one or more of the files in `web/thesauruses/_meta` into the language subdirectory. From there you should be good to implement those features in that concept file!
+After you save that file, go make a new subdirectory with same name in `web/thesauruses/`. From there, copy one or more of the files in `web/thesauruses/_templates` into the language subdirectory. From there you should be good to implement those features in that concept file!
+
```

Comment:
We didn't end up adding a `web/thesauruses/_templates` directory. Is this a leftover from your local machine? The templates are now generated on the website.

Reply:
Nice catch!